src/packages/client/src/runtime/dmmf.ts

```diff
@@ -68,8 +90,12 @@ export class DMMFClass implements DMMF.Document {
       type.fieldMap = keyBy(type.fields, 'name')
     }
   }
-  protected resolveInputTypes(types: DMMF.InputType[]) {
-    for (const type of types) {
+  protected resolveInputTypes() {
+    const inputTypes = this.inputObjectTypes.prisma
+    if (this.inputObjectTypes.model) {
+      inputTypes.push(...this.inputObjectTypes.model)
```

Comment:
Maybe we could use a faster way (based on https://www.measurethat.net/Benchmarks/Show/4223/0/array-concat-vs-spread-operator-vs-push)

```suggestion
      inputTypes.concat(this.inputObjectTypes.model)
```

Reply:
The option you suggest is not faster 🤷 
As far as I know, push with (...) is the fastest.
![image](https://user-images.githubusercontent.com/1094804/99046978-d04c1780-2593-11eb-98cd-cf497de2f855.png)
See https://dev.to/uilicious/javascript-array-push-is-945x-faster-than-array-concat-1oki#:~:text=concat%20performs%20at%200.40%20ops,is%20945x%20faster%20than%20concat%20!

Reply:
In their example the spread operator is faster, but afaik that's only for very small arrays.